cmd/endpoint.go

```diff
@@ -172,6 +177,11 @@ func (endpoints EndpointList) IsHTTPS() bool {
 	return endpoints[0].IsHTTPS()
 }
 
+// IsHTTP - returns true if insecure for URLEndpointType.
+func (endpoints EndpointList) IsHTTP() bool {
```

Comment:
`IsHTTP()` methods are not necessary.  You could use `!IsHTTPS()` directly.

Reply:
Now I remember why we removed this method in the past.